website/content/docs/agent-and-proxy/agent/generate-config/index.mdx

```diff
@@ -40,13 +40,13 @@ unless a path is specified as an argument.
 Before generating a configuration file, let's insert a secret `foo`:
 
 ```shell-session
-$ vault kv put -mount=secret foo user="admin" password="s3cr3t"
+$ openbao kv put -mount=secret foo user="admin" password="s3cr3t"
```

Comment:
The binary/CLI for `openbao` is just `bao`, I believe.  We'll need to change these on all the pages.

Reply:
:+1: and I can confirm this change has been made to the build system already :-)

Reply:
That's good to know 👍 ,
I'll change the name of the CLI accordingly